xsens-tf-bridge/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 2.8.9)
+find_package(YARP REQUIRED)
+include_directories(${YARP_INCLUDE_DIRS})
+
+find_package(YCM REQUIRED)
+include(YCMDefaultDirs)
```

Comment:
I don't think we are using the `YCMDefaultDirs` module for anything, please remove the include if that is the case.